### PR TITLE
Add HunyuanVideo ModelOpt FP8 diffusion support

### PR DESCRIPTION
**Pull request by BBuf**
## Summary

Add HunyuanVideo ModelOpt FP8 diffusion support and publish the SGLang-native transformer override under the `lmsys` Hugging Face org.

- add HunyuanVideo ModelOpt FP8 runtime support
- document the HunyuanVideo ModelOpt FP8 checkpoint flow in the diffusion quantization docs
- update the diffusion ModelOpt quant skill with the HunyuanVideo FP8 path
- add the HunyuanVideo ModelOpt FP8 case to the B200 diffusion CI set

## Published FP8 weights

- HunyuanVideo: https://huggingface.co/lmsys/hunyuanvideo-modelopt-fp8-sglang-transformer

The repo is intentionally clean: `README.md`, `config.json`, and `.safetensors` shards only.

## H100 Validation

Updated H100 validation used the `sglang-diffusion-benchmark-profile` HunyuanVideo command shape. This supersedes the earlier short-video validation.

Run setup:

- Host/GPU: H100 rank0, `CUDA_VISIBLE_DEVICES=0`
- Backend: `--backend=sglang`; logs show `Using pipeline from model_index.json: HunyuanVideoPipeline`, no diffusers fallback markers observed
- Model: `hunyuanvideo-community/HunyuanVideo`
- Prompt: `A cat and a dog baking a cake together in a kitchen. The cat is carefully measuring flour, while the dog is stirring the batter with a wooden spoon. The kitchen is cozy, with sunlight streaming through the window.`
- Skill preset args preserved: `--text-encoder-cpu-offload --pin-cpu-memory --num-frames=65 --width=848 --height=480 --num-inference-steps=30 --save-output --warmup --enable-torch-compile --seed=42`
- 5s adjustment: added `--fps=13`, so the output is exactly 65 frames / 13 fps = 5.000s
- FP8 delta: same command plus `--transformer-path lmsys/hunyuanvideo-modelopt-fp8-sglang-transformer`
- Profiler delta: same generation settings, replacing save/perf output with `--profile --num-profiled-timesteps=5 --no-save-output` and setting `SGLANG_DIFFUSION_TORCH_PROFILER_DIR`

Benchmark, warmup excluded:

| Metric | BF16 | FP8 | Delta | Speedup |
|---|---:|---:|---:|---:|
| E2E latency | 59.546 s | 54.748 s | -4.798 s (-8.1%) | 1.09x |
| Denoising stage | 42.542 s | 37.980 s | -4.562 s (-10.7%) | 1.12x |
| Avg denoise step | 1.4180 s | 1.2659 s | -0.1521 s | 1.12x |
| Decoding stage | 16.692 s | 16.458 s | -0.233 s (-1.4%) | 1.01x |
| Text encoding | 0.308 s | 0.306 s | -0.002 s (-0.7%) | 1.01x |

Profiler kernel share, 5 profiled denoise timesteps. Profiler timings include profiling overhead and are not used as benchmark latency numbers.

| Precision | Total CUDA op time | Top CUDA/kernel shares |
|---|---:|---|
| BF16 | 17.055 s | `cudaMemcpyAsync` 41.54%; FlashAttention 31.99%; BF16 GEMM `nvjet_tst_192x208_64x4_2x1_v_bz_coopB_bias_TNT` 9.77%; BF16 GEMM `nvjet_tst_192x208_64x4_1x2_h_bz_coopB_bias_TNT` 8.16%; BF16 GEMM `nvjet_tst_256x152_64x4_1x2_h_bz_coopA_bias_TNT` 2.11% |
| FP8 | 15.324 s | `cudaMemcpyAsync` 40.62%; FlashAttention 36.80%; FP8 Cutlass GEMM 12.83%; `triton_poi_fused_cat_gelu_view_0` 1.93%; `_static_quant_fp8` 1.37% |

## Local Validation After B200 CI Update

- `git diff --check` -> passed
- `python3 -m py_compile python/sglang/multimodal_gen/test/server/testcase_configs.py python/sglang/multimodal_gen/test/server/gpu_cases.py` -> passed
- `python3 -m black --check python/sglang/multimodal_gen/test/server/testcase_configs.py python/sglang/multimodal_gen/test/server/gpu_cases.py` -> passed
- `python3 -m ruff check --select=F401,F821 python/sglang/multimodal_gen/test/server/testcase_configs.py python/sglang/multimodal_gen/test/server/gpu_cases.py` -> passed

## B200 CI

Added to `ONE_GPU_MODELOPT_CASES` for `multimodal-gen-test-1-b200`:

- `hunyuanvideo_modelopt_fp8_t2v`

One caveat from the FP8 log: the CLI keeps the same offload flags as the skill preset, but the ModelOpt FP8 runtime currently forces `dit_cpu_offload` off while preserving layerwise offload behavior for restored FP8 tensor strides.

### CODE REVIEW

File: python/sglang/multimodal_gen/runtime/models/dits/qwen_image.py [hunk 1]
```diff
@@ -10,7 +10,6 @@
 import torch
```
Comment (mickqian) [inline python/sglang/multimodal_gen/runtime/models/dits/qwen_image.py:10]: duplicated contents?

**Review by mickqian — APPROVED** [review]

**Comment by BBuf** [comment]
Updated this PR to use the new clean `lmsys` ModelOpt diffusion repo.

- HunyuanVideo: https://huggingface.co/lmsys/hunyuanvideo-modelopt-fp8-sglang-transformer
- Added `hunyuanvideo_modelopt_fp8_t2v` to `ONE_GPU_MODELOPT_CASES` for `multimodal-gen-test-1-b200`.
- The shared lmsys collection is https://huggingface.co/collections/lmsys/diffusion-modelopt-69f06a1740c02269e36bf285

**Comment by BBuf** [comment]
https://github.com/sgl-project/sglang/actions/runs/25367205803/job/74399478580?pr=23199
